### ISSUE TITLE
Show neuroglancer state information in human readable format

### ISSUE THREAD

**Issue by austinhoag** [BrainSharer/database_portal#8]
It would be useful to have another column in the neuroglancer state table with a link to show the state information in human readable format. Right now, the user has to open the neuroglancer link to see the contents.

**Comment by eddyod** [comment]
This link points to the formatted URL:
https://www.brainsharer.org/brainsharer/admin/neuroglancer/neuroglancermodel/1/change/
Something like that?

**Comment by austinhoag** [comment]
Yes, something like that. I think we should parse the JSON into more human readable format though. Maybe break the state into sections, layers first. Each layer needs to have a description displayed at this page so that the user understands what it contains. I believe that's what the `comments` field of the `NeuroglancerModel` model is for, but correct me if I'm wrong. I think we can't assume that the layer name or path will contain enough information to describe the contents of that layer. 

Also, it was not clear to me that I had to click the animal name in the neuroglancer state table to get to that page. I think we should remove that link and perhaps ad another column in that table explicitly indicating that clicking the link will bring the user to a page where they can view the contents of the state. For example, "State description"
